hw3/01-characters.js

```diff
@@ -1,4 +1,44 @@
 // url for the Thrones API
 const url = 'https://thronesapi.com/api/v2/Characters';
 
-fetch(url);
+const parentDiv = document.createElement('div');
+parentDiv.classList.add('parent-div-01');
+
+const section = document.getElementById('section-01');
+section.appendChild(parentDiv);
+
+function createCharacterCard(character) {
+  const cardDiv = document.createElement('div');
+  cardDiv.classList.add('card-div-01');
+
+  const image = document.createElement('img');
+  image.classList.add('dp-01');
+  image.setAttribute('src', character.imageUrl);
+  image.setAttribute('alt', `Display picture of ${character.fullName}`);
+
+  const name = document.createElement('h2');
+  name.classList.add('full-name-01');
+  name.innerText = character.fullName;
```

Comment:
Why is innerText being used over textContent for this situation?

Reply:
Addressed.